Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -69,14 +69,22 @@
         {
             "name": "WebAppKind",
             "type": "pickList",
-            "label": "Web app kind",
+            "label": "App Type",
             "defaultValue": "app",
             "required": false,
-            "visibleRule": "WebAppName = invalidwebappname",
+            "options": {
```

Comment:
just a suggestion, Can we sort the app type by name or its popularity?
Can you please get it reviewed by PM?